test/e2e/configmap.go

```diff
@@ -181,7 +181,7 @@ var _ = Describe("ConfigMap", func() {
 		// Kubelet projects the update into the volume and the container picks
 		// it up. This timeout is based on the default Kubelet sync period (1
 		// minute) plus additional time for fudge factor.
-		const podLogTimeout = 90 * time.Second
+		const podLogTimeout = 150 * time.Second
```

Comment:
Is there a way to link this directly to the sync interval, so it floats against it?

Also, it should be `≤2*interval`, right?